modules/logging-release-notes-5-6-16.adoc

```diff
@@ -0,0 +1,12 @@
+// module included in /logging/logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-16_{context}"]
+= Logging 5.6.16
+This release includes link:https://access.redhat.com/errata/RHSA-2024:0695[Logging Bug Fix 5.6.16]
+
+[id="logging-release-notes-5-6-16-bug-fixes"]
+== Bug fixes
+* Before this update, when configured to read a custom S3 Certificate Authority the Loki Operator would not automatically update the configuration when the name of the ConfigMap or the contents changed. With this update, the Loki Operator is watching for changes to the ConfigMap and automatically updates the generated configuration. (link:https://issues.redhat.com/browse/LOG-4967[LOG-4967])
+
+[id="logging-release-notes-5-6-16-CVEs"]
+== CVEs
```

Comment:
There is one CVE bug in 5.6.16 AD, but the CVE is not listed here.

Reply:
This is because it is listed as private and IIRC we don't mention those?!?

Reply:
@periklis is correct. The OCP repo will not allow a PR build to pass if a link to a private issue is included.